### PR TITLE
Changed Glip to RingCentral Classic

### PR DESCRIPTION
**Pull request by the-jay-schwartz**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/ringcentral-classic.rb [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
+  sha256 'f7da52b2c132240a93e3fbff0bd92a0276288703ea2e2a7b9d54474be3f489cc'
 
-  # downloads.ringcentral.com/glip/rc/ was verified as official when first introduced to the cask
-  url "https://downloads.ringcentral.com/glip/rc/#{version}/mac/RingCentral-#{version}.dmg"
-  appcast 'https://macupdater.net/cgi-bin/check_urls/check_url_redirect.cgi?url=https://downloads.ringcentral.com/glip/rc/GlipForMac'
-  name 'Glip'
-  homepage 'https://glip.com/'
+  url "https://downloads.ringcentral.com/glip/rc/#{version}/mac/RingCentral%20Classic-#{version}.dmg"
+  name 'RingCentral Classic'
+  homepage 'https://www.ringcentral.com/apps/rc-classic'
```
Comment (vitorgalvao) [inline Casks/ringcentral-classic.rb:7]: This page still shows `20.2.12`. Got a better one?

Comment (JayTheMarketer) [reply Casks/ringcentral-classic.rb:7]: Yeah -- all the downloads are here: https://support.ringcentral.com/download.html

Comment (vitorgalvao) [reply Casks/ringcentral-classic.rb:7]: That’s not really a homepage, though. This one will have to do for now. Will this app be updated further, even?

Comment (JayTheMarketer) [reply Casks/ringcentral-classic.rb:7]: I don't believe so. To the best of my understanding, updates will not continue for Classic.

**Comment by vitorgalvao** [comment]
For future reference, before rename:

```
glip
30 days: 6 (#3740)
90 days: 40 (#2608)
365 days: 80 (#3256)
Age: 1025 days (added 2017, July 19)
```

**Pull request merged**